Assets/MixedRealityToolkit/Utilities/MRTKServicesUtils.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Boundary;
+using Microsoft.MixedReality.Toolkit.Diagnostics;
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.SpatialAwareness;
+using Microsoft.MixedReality.Toolkit.Teleport;
+
+namespace Microsoft.MixedReality.Toolkit
+{
+    /// <summary>
+    /// Utility class to easily access references to runtime Mixed Reality Toolkit Services
+    /// </summary>
+    public static class MRTKServicesUtils
+    {
+        private static IMixedRealityInputSystem inputSystem = null;
+
+        /// <summary>
+        /// The active instance of the input system
```

Comment:
As long as we're adding this new stuff, it might be worth calling out explicitly that this will return a cached value, and that if the input system is ever torn down or destroyed, this would still technically return that object (until the InvalidateReferences() thing is called).

It might be worth either having:

1) Some doccomment remark to the class saying that this is the general behavior of the class.
2) Having a per-function remark (this increases copy paste, but at the same time increases the chances that folks will actually read it when they're doing autocomplete stuff)

Reply:
Registrars could possibly trigger an invalidate or we could add an event to signal the cache to clear. Would love to hear thoughts on these options.

Reply:
Yes, I thought of this too. The original approach people have been taking in code has the same problem. I'm not sure of the best way to approach on how to notify listeners of an invalidated service